distance

```diff
@@ -1,19 +1,22 @@
-def dist x,y
-  diff1=x[0].to_f - x[1].to_f
-  diff2=y[0].to_f - y[1].to_f
+class Point
```

Comment:
extract Point class in point.rb
and right a test for it. Look up https://relishapp.com/rspec/docs/gettingstarted